Classes/Processor/ImageProcessor.php

```diff
@@ -27,6 +28,13 @@ public function __construct(ContentObjectRenderer $contentObject)
     public function renderImage(FileInterface $file, array $conf = []): array
     {
         $defaultImageResource = $this->contentObject->getImgResource($file, $conf);
+        if (is_null($defaultImageResource[3])) {
```

Comment:
```suggestion
        if (!isset($defaultImageResource[3])) {
```

`$defaultImageResource[3]` might not be set at all.

Reply:
I have refactored the condition